templates/footer.hbs

```diff
@@ -0,0 +1,24 @@
+<footer class="container--color">
+    <div class="footer">
+        <div class="footer__logo">
+            <a href="/index.html"><img src="/images/transformasion-logo-large.png" alt=""></a>
+            <a href=""></a>
+        </div>
+            <ul class="footer__nav">
+                {{#each navLinks}}
+                    <li class="footer__nav--space">
+                        {{{module this}}}
+                    </li>
+                {{/each}}
+            </ul>
+            <hr class="footer__line"></hr>
+            <ul class="footer__social">
+                {{#each socialLinks}}
+                    <li>
+                        {{{module this}}}
+                    </li>
+                {{/each}}
+            </ul>
+        <img class="footer__copyright" src="/images/CopyrightIcon.png" alt="">
```

Comment:
Here you need to use the variable coming from Contentful, instead of an image
![image](https://user-images.githubusercontent.com/11170707/130153413-7b702b82-15da-4015-9b4a-c415d58653f6.png)